query-engine/connector-test-kit-rs/README.md

```diff
@@ -3,12 +3,13 @@ The test kit is a (currently incomplete) port of the Scala test kit, located in
 It's fully focused on integration testing the query engine through request-response assertions.
 
 ## Test organization
-The test kit is a combination of three crates, from which two are "lower level" crates that are only required to make it work, whereas only one is important if you only want to author tests. The three-crate approach is required to prevent circular dependencies between the crates (arrow = depends-on):
+
+The test kit is a combination of three crates, from which two are "lower level" crates that are only required to make it work, whereas only one is important if you only want to author tests.
 ```
                ┌────────────────────┐
-           ┌───│ query-engine-tests │───┐
```

Comment:
shouldn't be the other one the arrow removed? Aren´t we using the macros directly from the tests?

Reply:
True, good catch